File: src/main/java/com/graqr/threshr/model/redsky/products/MerchandiseClassification.java
package com.graqr.threshr.model.redsky.products;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record MerchandiseClassification(
        @JsonProperty("class_id") Long classId,
        @JsonProperty("department_id") Long departmentId,
        @JsonProperty("department_name") String departmentName) {
}

File: src/main/kotlin/com/graqr/threshr/model/redsky/product/MerchandiseClassification.kt
package com.graqr.threshr.model.redsky.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class MerchandiseClassification(
    @param:JsonProperty("class_id") val classId: Long,
    @param:JsonProperty("department_id") val departmentId: Long,
    @param:JsonProperty("department_name") val departmentName: String
) {}
